### PR TITLE
[bug]: fix the issue of continue shopping button background color and hover color

### LINKED ISSUE #1306: [bug]: continue shopping button background color and hover color
**Describe the bug**
Continue shopping button background color and hover color fixed

**To Reproduce**
1. Minicart(Empty) check continue shopping button

**Expected behavior**
expected1
expected2

**Actual behavior**
actual1
actual2

**Please let us know what packages this bug is in regards to:**
- [x] `venia-concept`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

### PR DESCRIPTION
**Pull request by speedy008**
## Related Issue
Closes #1306 #1329   - continue shopping button background color and hover color 

## Description

Continue shopping button background color and hover color fixed

## Verification Steps

1. Minicart(Empty) check continue shopping button

## How Have YOU Tested this?

1. Tested into my PWA-studio latest instance 

## Screenshots / Screen Captures (if appropriate):
1. Actual Result of Continue shopping button
actual1
actual2

1. Expected Result of Continue shopping button.
expected1
expected2

## Proposed Labels for Change Type/Package

- [ ] major (e.g x.0.0 - a breaking change)
- [ ] minor (e.g 0.x.0 - a backwards compatible addition)
- [x] patch (e.g 0.0.x - a bug fix)

## Checklist:

- [x] I have updated the documentation accordingly, if necessary.
- [x] I have added tests to cover my changes, if necessary.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1264) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1264) before we can accept your contribution.<br/>

File: packages/venia-concept/src/components/MiniCart/emptyMiniCart.css [hunk 1]
```diff
@@ -16,9 +16,10 @@
...
     composes: root from '../Button/button.css';
     color: white;
-    background-color: rgb(var(--venia-teal));
+    background-color: rgb(var(--color));
 }
 
 .continue:hover {
-    color: rgb(var(--venia-teal));
+    color: white;
+    background-color: rgb(var(--venia-teal));
```
Comment (sirugh) [inline packages/venia-concept/src/components/MiniCart/emptyMiniCart.css]: @soumya-ashok what is the desired state for hover? [The mocks](https://magento.invisionapp.com/share/WQN5F7BYBPG#/screens/316856656) show the black background but I don't think inVision lets you do hover states in mocks (or I just cant find one).

**Comment by sirugh** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  node` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh
Error:  Error: Could not find a `dangerfile.js` or `dangerfile.ts` in the current working directory.
    at Object.dangerfilePath (/codebuild/output/src402962295/src/github.com/magento-research/pwa-studio/node_modules/danger/distribution/commands/utils/fileUtils.js:26:11)
    at Object.<anonymous> (/codebuild/output/src402962295/src/github.com/magento-research/pwa-studio/node_modules/danger/distribution/commands/danger-runner.js:92:42)
    at step (/codebuild/output/src402962295/src/github.com/magento-research/pwa-studio/node_modules/danger/distribution/commands/danger-runner.js:33:23)
    at Object.next (/codebuild/output/src402962295/src/github.com/magento-research/pwa-studio/node_modules/danger/distribution/commands/danger-runner.js:14:53)
    at fulfilled (/codebuild/output/src402962295/src/github.com/magento-research/pwa-studio/node_modules/danger/distribution/commands/danger-runner.js:5:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)

```

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a2c31ec216caa46f7c61b77bbeb1ffae1106809c
</p>

**Comment by sirugh** [comment]
@speedy008 can you please open an issue describing this work? Since it has to do with UX it will need to be reviewed by our internal UX team and that discussion should happen in the issue rather than in a PR. Once you do open the issue, link it here with the text 'closes #<issue_number' in the description.

Cheers!

**Comment by speedy008** [comment]
@sirugh I have created an issue for the same

File: packages/venia-concept/src/components/MiniCart/emptyMiniCart.css (hunk 1 above)
Comment (soumya-ashok) [reply packages/venia-concept/src/components/MiniCart/emptyMiniCart.css]: @sirugh - The general pattern Jimmy and I decided on is that the button is dark gray in its static state and turns to teal in the hover or focused state.

**Comment by supernova-at** [comment]
I have added the `hold` label to this PR and #1367. They solve the same problem in different ways so we just need to decide which solution we want to move forward with.

Further, this work needs to be incorporated into the existing #1266 MiniCart refactor, or added as a fast follow.

**Pull request merged**